.github/workflows/zeebe-ci.yml

```diff
@@ -186,6 +186,57 @@ jobs:
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+  strace-tests:
+    # Zeebe tests requiring strace
+    name: Strace Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
Probably from copy&paste, let's use a timeout closer to the actual runtime in the range of 5-10 minutes.

Reply:
Ok! We can also settle on 15min orso but 30min seems too much..

Reply:
It [took](https://github.com/camunda/camunda/actions/runs/11050067884/job/30697060018) ~14min, I prefer to set timeout to 20min to have some margin.